### PR TITLE
feat(applications): New libary for ui-forms and new form for national registry data

### PR DESCRIPTION
**Pull request by egill15**
# UI-FORMS

https://app.clickup.com/t/2y2yfpw

## What
Form for applicant data from national registry

## Why
Reduce duplicated code

## Screenshots / Gifs

image

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/application/ui-forms/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Application UI Forms
```
Comment (eirikurn) [inline libs/application/ui-forms/README.md:1]: Should describe the purpose of the library here. Also good to consider the gitbook navigation tree, like `<!-- gitbook-navigation: "UI Forms" -->`

File: libs/application/ui-forms/src/lib/applicantInformationSection/messages.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  general: defineMessages({
+    title: {
+      id: 'ais.application:applicantInfo.general.title',
+      defaultMessage: 'Upplýsingar um þig',
+      description: 'Information about you',
+    },
+  }),
+  labels: defineMessages({
+    name: {
+      id: 'ais.application:applicantInfo.labels.name',
```
Comment (eirikurn) [inline libs/application/ui-forms/src/lib/applicantInformationSection/messages.ts:13]: Needs a different namespace. Either one for all shared ui-forms or one per each.

File: libs/application/ui-forms/project.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  "tags": [],
+  "targets": {
+    "lint": {
+      "executor": "@nrwl/linter:eslint",
+      "outputs": ["{options.outputFile}"],
+      "options": {
+        "lintFilePatterns": ["libs/application/ui-forms/**/*.{ts,tsx,js,jsx}"]
+      }
+    },
+    "test": {
```
Comment (eirikurn) [inline libs/application/ui-forms/project.json:14]: Needs a translation script.

**Review by eirikurn — APPROVED** [review]
Nice! Approving with minor comments.

**Review by Thorrik58 — APPROVED** [review]
LGTM

**Pull request merged**